shared/src/main/scala/com/nawforce/pkgforce/documents/MetadataValidator.scala

```diff
@@ -270,4 +270,31 @@ class MetadataValidator(logger: IssuesManager, namespace: Option[Name], isGulped
         })
       })
   }
+
+  /** Validate case sensitivity for metadata directory names */
+  private def validateCaseSensitivity(
+    documents: List[PathLike],
+    expectedDirectoryName: String
+  ): Unit = {
+    documents.foreach { document =>
+      // Find the parent directory that should match expectedDirectoryName
+      val parentDir = document.parent.basename
+
+      if (
+        parentDir.toLowerCase == expectedDirectoryName.toLowerCase && parentDir != expectedDirectoryName
```

Comment:
I don't mind it explicit like this, but I think we've used `equalsIgnoreCase()` before. Perhaps same treatment for those case statements in MetadataDocument.

Reply:
Done